src/script/descriptor.cpp

```diff
@@ -524,6 +540,21 @@ class RawDescriptor final : public DescriptorImpl
 public:
     RawDescriptor(CScript script) : DescriptorImpl({}, {}, "raw"), m_script(std::move(script)) {}
     bool IsSolvable() const final { return false; }
+
+    Optional<OutputType> GetOutputType() const override
+    {
+        CTxDestination dest;
+        ExtractDestination(m_script, dest);
+        switch(dest.which()) {
+            case 1:
```

Comment:
Please use `PKHash` (etc) here for readability.

Reply:
Done